x/concentrated-liquidity/README.md

```diff
@@ -1712,6 +1712,42 @@ creates a position with amounts such that the current tick is 155 between ticks
 
 Note, that the current tick of 155 is defined inside the bucket over a range of 100 to 200.
 
+- `Initialized Tick` - a tick at which LPs can provide liquidity. Some ticks cannot be
+initialized due to tick spacing. `MinCurrentTick` is an exception due to being 1 tick below
+`MinInitializedTick`. Only initialized ticks are crossed during a swap (see "Crossing Tick")
+for details.
+
+- `MinInitializedTick` - the minimum tick at which a position can be initialized. When this tick is
+crossed, all liquidity is consumed at the tick ends up on `MinCurrentTick`. At that point, there
+is no liquidity and the pool is in no bucket. To enter the first bucket, a swap right must be done
+to cross the next initialized tick and kick in the liquidity. If at least one full range position is
+defined, `MinInitializedTick` will be the first such tick.
+
+- `MinCurrentTick` - is the minimum value that a current tick can take. If we consume all liquidity and
+cross the min initialized tick, our current tick will equal to MinInitializedTick - 1 (MinCurrentTick)
+with zero liquidity. However, note that this `MinCurrentTick` cannot be crossed. If current tick equals
```

Comment:
nit: if we're going in the positive tick direction, then doesn't being in the `MinCurrentTick` imply that it was "crossed"? Feel free to disregard if this is incompatible with the tick crossing definition below

Reply:
Yes, yhe upper bound was crossed which is `MinInitializedTick` and equals to `MinCurrentTick + 1`. The upper bound is exclusive so this is correct.

If we go left-to-right, we do not cross the "current tick" value. However, we do cross the next initialized tick which is `MinInitializedTick` as long as there is at least one full range position.

There is no lower bound to cross since `MinCurrentTick` is the minimum possible value that cannot be initialized. I attempt to explain this edge case under the "Crossing Tick" definition.

Let me know if this helps